### ISSUE TITLE
Feature Request: Support for APCA as included in WACAG 3 (draft)

### ISSUE THREAD

**Issue by meetbryce** [SiTaggart/AreMyColorsAccessible#1406]
https://github.com/Myndex/apca-w3/

Happy to be helpful if a PR is welcomed.

**Comment by SiTaggart** [comment]
Yeah, ultimately I've been holding off until it's not "draft" mainly because if you implement it now and get audited, you won't be accessible.

I might look at being able to switch between the two, but I am cautious about giving people false hope and information.
